src/nvim/terminal.c

```diff
@@ -217,10 +216,14 @@ Terminal *terminal_open(TerminalOptions opts)
   vterm_set_utf8(rv->vt, 1);
   // Setup state
   VTermState *state = vterm_obtain_state(rv->vt);
+  // Only relevant in terminal true color mode
+  if (true_color) {
```

Comment:
Yeah, I'm just trying to think of where to put these options, since they are starting to take root.